patch-image.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/bash
+set -ex
+
+# PATCH argument is in the form "project refs" where refs is a list of
+# comma-separated gerrit refs, fo example refs/changes/67/759567/1
+PROJ_NAME=$(echo $1 | cut -d " " -f1)
+REFSPECS=$(echo $1 | cut -d " " -f2)
+
+cd /tmp
+git clone "https://opendev.org/openstack/$PROJ_NAME"
+cd "$PROJ_NAME"
+for REFSPEC in $REFSPECS; do
```

Comment:
The comment says it's a comma-separated list, but I don't see us splitting on commas anywhere?